lib/tupelo.js

```diff
@@ -552,11 +552,11 @@ const setDataTransaction = (path, value) => {
  *
  * @return SetOwnerTransaction
  */
-const setOwnerTransaction = (authentication) => {
+const setOwnerTransaction = (newOwnerKeys) => {
```

Comment:
The param name here was mismatched with [line 551](https://github.com/quorumcontrol/tupelo.js/pull/48/commits/12d09894c597ec422837cf419598676cadaf60a2#diff-72abe2583747e0007b0681ac8126741aR551)